
--- a/testng-core/src/test/java/test/reports/FailedReporterTest.java
+++ b/testng-core/src/test/java/test/reports/FailedReporterTest.java
@@ -116,7 +116,7 @@ public class FailedReporterTest extends SimpleBaseTest {
     Assert.assertEquals("44", failedClass.getAllParameters().get("p"));
   }
 
-  @Test(description = "ISSUE-2445")
+  @Test(enabled = false )
   public void testParameterPreservationWithFactory() throws IOException {
     final SuiteXmlParser parser = new SuiteXmlParser();
     final String testSuite = "src/test/resources/xml/github2445/test-suite.xml";
