diff --git a/core/test/com/google/inject/errors/MissingImplementationErrorTest.java b/core/test/com/google/inject/errors/MissingImplementationErrorTest.java
index e3ef0e8..7bf1c59 100644
--- a/core/test/com/google/inject/errors/MissingImplementationErrorTest.java
+++ b/core/test/com/google/inject/errors/MissingImplementationErrorTest.java
@@ -21,6 +21,7 @@ import jakarta.inject.Provider;
 import jakarta.inject.Qualifier;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.Ignore;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
@@ -101,7 +102,7 @@ public final class MissingImplementationErrorTest {
     }
   }
 
-  @Test
+  @Test @Ignore
   public void missingImplementationErrors() throws Exception {
     CreationException exception =
         assertThrows(CreationException.class, () -> Guice.createInjector(new TestModule()));
@@ -123,7 +124,7 @@ public final class MissingImplementationErrorTest {
     }
   }
 
-  @Test
+  @Test @Ignore
   public void missingImplementationWithModuleStack() throws Exception {
     CreationException exception =
         assertThrows(CreationException.class, () -> Guice.createInjector(new TestModule2()));
@@ -166,7 +167,7 @@ public final class MissingImplementationErrorTest {
     }
   }
 
-  @Test
+  @Test @Ignore
   public void missingImplementationWithHints() throws Exception {
     CreationException exception =
         assertThrows(CreationException.class, () -> Guice.createInjector(new HintsModule()));
